proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -77,7 +77,7 @@ message ForwardMsg {
     // Other messages.
     Navigation navigation = 23;
     PageNotFound page_not_found = 15;
-    PagesChanged pages_changed = 16;
+    PagesChanged pages_changed = 16 [deprecated = true];
```

Comment:
nit: it's probably fine to use `deprecated`, but we might want to validate this somehow with Notebooks/SiS if it breaks any part of what we consider as compatibility. I already discussed this a few days ago with @raethlein (see the tech debt item [here](https://www.notion.so/snowflake-corp/Use-deprecated-protobuf-field-option-for-deprecated-fields-1937170bb41680b19a29ea54e227611f)) but I think we haven't gotten a conclusion on this